internal/provider/provider.go

```diff
@@ -109,8 +115,13 @@ func (p *ProwlarrProvider) Configure(ctx context.Context, req provider.Configure
 	} else {
 		key = data.APIKey.ValueString()
 	}
-
-	if key == "" {
+	var authorization string
+	if data.Authorization.IsNull() {
+		authorization = os.Getenv("PROWLARR_AUTHORIZATION")
+	} else {
+		authorization = data.Authorization.ValueString()
+	}
+	if key == "" || authorization == ""{
```

Comment:
Either one has to be populated. I don't think you can communicate to prowlarr without authorization?

Reply:
I may be misunderstanding, but I believe it currently says if either `key` or `authorization` is empty then it will fail. It should be valid for at least one of them to be empty, but not both.

Reply:
Oh, you're right. Fixing now.